### PR TITLE
Build/deploy the projection claims app as part of EEI prereqs.sh

### PR DESCRIPTION
**Pull request by DanRoseus**
Summary of changes:
- Moved the pipeline yamls out of QuoteLifecycleSimulator into a build dir
- Moved the pipeline scripting out of prereqs.sh into build.sh to make it easier to rebuild
- Add parallel build/deploy of ProjectionClaims app to pipeline
- Removed the services accounts/role bindings/roles, now runs with the default pipeline service account
- Improved labelling of k8s resources
- Updated the password for the replication user so prereqs.sh can be run multiple times

### CODE REVIEW

File: EventEnabledInsurance/build/build.sh [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+echo "INFO: Branch: '$BRANCH'"
+echo "INFO: TKN: '$TKN'"
+
+echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
+
+echo "INFO: Creating pvc for EEI apps in the '$namespace' namespace"
+if oc apply -n $namespace -f $CURRENT_DIR/pvc.yaml; then
+  echo -e "\n$tick INFO: Successfully created the pvc in the '$namespace' namespace"
+else
+  echo -e "\n$cross ERROR: Failed to create the pvcin the '$namespace' namespace"
```
Comment (Kinshuk1993) [inline EventEnabledInsurance/build/build.sh:81]:
```suggestion
  echo -e "\n$cross ERROR: Failed to create the pvc in the '$namespace' namespace"
```

File: EventEnabledInsurance/build/build.sh [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
+
+echo "INFO: Creating the pipeline to build and deploy the EEI apps in '$namespace' namespace"
+if cat $CURRENT_DIR/pipeline.yaml |
+  sed "s#{{NAMESPACE}}#$namespace#g;" |
+  sed "s#{{FORKED_REPO}}#$REPO#g;" |
+  sed "s#{{BRANCH}}#$BRANCH#g;" |
+  oc apply -n ${namespace} -f -; then
+    echo -e "\n$tick INFO: Successfully applied the pipeline to build and deploy the EEI apps in in '$namespace' namespace"
```
Comment (Kinshuk1993) [inline EventEnabledInsurance/build/build.sh:92]:
```suggestion
    echo -e "\n$tick INFO: Successfully applied the pipeline to build and deploy the EEI apps in '$namespace' namespace"
```

File: EventEnabledInsurance/build/build.sh [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+else
+  echo -e "\n$cross ERROR: Failed to apply the pipeline to build and deploy the EEI apps in in '$namespace' namespace"
```
Comment (Kinshuk1993) [inline EventEnabledInsurance/build/build.sh:94]:
```suggestion
  echo -e "\n$cross ERROR: Failed to apply the pipeline to build and deploy the EEI apps in '$namespace' namespace"
```

File: EventEnabledInsurance/prereqs.sh [hunk 4]
```diff
@@ -325,84 +258,13 @@ $TKN clustertask ls | grep git-clone
...
-
-if oc delete pipelinerun -n $namespace $(oc get pipelinerun -n $namespace | grep $PIPELINERUN_UID | awk '{print $1}'); then
-  echo -e "$tick INFO: Deleted the pipelinerun with the uid '$PIPELINERUN_UID'"
 else
-  echo -e "$cross ERROR: Failed to delete the pipelinerun with the uid '$PIPELINERUN_UID'"
-fi
-
-echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
+  echo -e "\n$tick INFO: Successfully built and deplopyed the EEI apps in the '$namespace' namespace"
+fi #create-ace-config.sh
```
Comment (Kinshuk1993) [inline EventEnabledInsurance/prereqs.sh:267]:
```suggestion
fi #build/build.sh
```

**Review by Kinshuk1993 — COMMENTED** [review]
some minor suggestions

**Review by Kinshuk1993 — APPROVED** [review]
looks good!

**Review by laurensks — APPROVED** [review]
Looks good!

**Pull request merged**